uint8 FLIGHT_MODE_CHANGE = 1
uint8 VEL_FLIGHT_MODE_COMMAND = 2

uint8 command_mode
float32 target_vel_x
float32 target_vel_y
float32 target_vel_psi
float32 target_pos_psi
